specification/dimensions/chargesubcategory.md

```diff
@@ -0,0 +1,63 @@
+# Charge Subcategory
+
+Charge Subcategory indicates what kind of usage or adjustment the row represents. Charge Subcategory is a supplemental detail to the [Charge Type](#chargetype). It provides additional context to describe the primary category of charge.
+
+This linkage to the parent Charge Type means that for every entry under Charge Type, there can be a corresponding Charge Subcategory that further refines the nature of the charge. It's a nested level of detail that allows users to see not just what type of charge was incurred. Current sub-categorization currently applies to Charge Type values 'Usage' and 'Adjustment'. Support for others will be added as needed.
+
+When Charge Type is "Usage" and the charge is related to a commitment, the Charge Subcategory indicates whether the row represents committed usage or is an amortized charge for the unused portion of the commitment. Charge Subcategory is commonly used for scenarios like calculating commitment utilization when ChargeType is "Usage".
+
+When Charge Type is "Adjustment", the Charge Subcategory indicates what kind of after-the-fact adjustment the record represents and is commonly used to identify changes like credits and refunds.
+
+ChargeSubcategory MUST follow the requirements listed below:
+
+- The ChargeSubcategory MUST be present in the billing data.
+- ChargeSubcategory is of type String and MUST be one of the allowed values.
+- ChargeSubcategory MUST NOT be null or empty when ChargeType is "Usage" and the charge is covered by a commitment.
+  - When a usage charge is covered by a commitment, ChargeSubcategory MUST be "Used Commitment".
+  - When a commitment is not used within the committed period, ChargeSubcategory MUST be "Unused Commitment" for the unused usage charge.
```

Comment:
```suggestion
  - When a commitment is not completely used within the committed period, ChargeSubcategory MUST be "Unused Commitment" for the unused usage charge.
```

Reply:
@aqu-erp can we say instead of not completely used, say when a commitment is not used (Fully or partially)....